fitting/fitScanData.py

```diff
@@ -313,7 +314,7 @@ def fit(self, debug=False):
                     
                     if debug:
                         if self.isVFAT3:
-                            print "| %i | %i | %i | %i | %f | %f | %f | %f | %f | %f | %f | %f | %f |"%(
+                            print("| {0} | {1} | {2} | {3} | {4} | {5} | {6} | {7} | {8} | {9} | {10} | {11} | {12} |".format(
```

Comment:
It wasn't my intention with my comment to have you do **all** the conversions, (just the ones that were in code you added/modified), but it's a good thing to have done anyway :-)
However, here does the formatting stay constant?
I suppose it can guess at the type, just need to make sure there isn't any implicit type conversion happening during the formatting...

Reply:
I ended up doing a replace for the files I edited, and changed up the numbers when need be! I can add types to these just to be on the safe side